core/src/main/java/org/lflang/diagram/synthesis/styles/LinguaFrancaStyleExtensions.java

```diff
@@ -76,6 +78,12 @@ public class LinguaFrancaStyleExtensions extends AbstractSynthesisExtensions {
   @Inject @Extension private KPolylineExtensions _kPolylineExtensions;
   @Extension private KRenderingFactory _kRenderingFactory = KRenderingFactory.eINSTANCE;
 
+  public static final String SELECTION_HIGHLIGHTING_COLOR_LABEL = "Selection Coloring";
+
+  public static final SynthesisOption SELECTION_HIGHLIGHTING_COLOR =
+      SynthesisOption.createCheckOption(SELECTION_HIGHLIGHTING_COLOR_LABEL, false)
+          .setCategory(LinguaFrancaSynthesis.APPEARANCE);
+
```

Comment:
Currently, the new synthesis option is in the appearance category and its default value is false such that it will not break things but can be enabled.